CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ## Enhancements
 
 - Added `NormalizeSchemas` configuration property to the Avro, Json and Protobuf serdes.
+- Upgraded `NJsonSchema` to v10.6.3
+- Added `LatestCompatStrict` configuration property to JsonSerializerConfig to check the compatibility with latest schema
```

Comment:
LatestCompatibilityStrict

Reply:
In Java client it is referenced as LatestCompatStrict. So went with same field name.

Reply:
this just an internal variable name. what matters is how it is exposed.